examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -44,6 +44,39 @@
     "BATCH_SIZE = 64"
```

Comment:
<p>You save a model, never load it, and in the end you show one more way to save a model. Does it make sense?</p><p>Make a block that explains different ways to save/load/import models, or make sense of it in some other way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>